packages/client/hmi-client/src/workflow/tera-operator.vue

```diff
@@ -1,8 +1,16 @@
 <template>
-	<main :style="nodeStyle" ref="operator">
+	<main
+		:style="nodeStyle"
+		ref="operator"
+		@mouseenter="interactionStatus |= OperatorInteractionStatus.Hover"
+		@mouseleave="interactionStatus &= ~OperatorInteractionStatus.Hover"
```

Comment:
I was thinking to create utility functions.

Reply:
Do you mean functions for adding/removing bitmasks?

Reply:
Yes and test: `isHover()`, or `addHover()`, `removeHover()`, etc.